bokeh/tests/test_properties.py

```diff
@@ -317,14 +317,28 @@ class IsDelegate(HasProps):
 
         class IncludesDelegateWithPrefix(HasProps):
             z = Include(IsDelegate, use_prefix=True)
-            z_y = String("world") # override the Include
+            z_y = Int(57) # override the Include
 
         class IncludesDelegateWithoutPrefix(HasProps):
             z = Include(IsDelegate, use_prefix=False)
-            y = String("world") # override the Include
+            y = Int(42) # override the Include
+
+        class IncludesDelegateWithoutPrefixUsingOverride(HasProps):
+            z = Include(IsDelegate, use_prefix=False)
+            y = Override(default="world") # override the Include changing just the default
```

Comment:
conceptually I think Include is a mixin, but we have to do it he Include way so we can set a prefix, so that's why this seems logical to me.